src/index.css

```diff
@@ -1,3 +1,15 @@
+@import url('https://fonts.googleapis.com/css2?family=Pacifico&display=swap');
+
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
 :root {
   font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
   line-height: 1.5;
```

Comment:
You have made a great attempt to style the navigation but in the process, you changed the look of the login/ register:
**`from this:`**
![Screenshot from 2023-08-19 02-29-24](https://github.com/stephenkati/gaming_console_reservation_front_end/assets/47294794/903e82b9-8a1e-459b-8f78-219a5bdba523)

**`the resulting login/ register is`:**

![Screenshot from 2023-08-19 02-33-00](https://github.com/stephenkati/gaming_console_reservation_front_end/assets/47294794/51e26362-3727-463b-90f0-2792c0e21079)
![Screenshot from 2023-08-19 02-24-30](https://github.com/stephenkati/gaming_console_reservation_front_end/assets/47294794/6aa95afc-618b-4430-a06c-d79a94f0eed4)


Ensure the login/register is the `same size` and at the `center` for all devices

Reply:
I have edited the layout for the `sign_in` and `login` accordingly.

## OPTIONAL SUGGESTION
I don't know if this is the final look of the forms because, in the requirements, we are supposed to design the reservation form based on the ["Book a vespa test-ride"](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign/modules/173005583) design. 

So, I thought maybe we could follow the same design for all app forms, just for consistency.

![login-signup-form-design](https://github.com/stephenkati/gaming_console_reservation_front_end/assets/90119799/d4eae061-79b4-409e-be2b-3c266da0664c).